pyproject.toml

```
@@ -0,0 +1,60 @@
+[project]
+name = "unitycatalog-function"
```

```suggestion
name = "unitycatalog-functions"
```

should this be plural?

---

btw is this a name space package?

---

Yea this name is pending to be changed, I'm waiting for Xiangrui's response :)